src/graphics.h

```diff
@@ -18,6 +18,7 @@ struct graphics_color {
 int graphics_width();
 int graphics_height();
 
+void graphics_line( int x1, int y1, int x2, int y2, struct graphics_color c );
```

Comment:
I looked at Kyle's pr and put a Doxygen standard on the Coding Style standards google doc.